channels/unsorted.m3u

```diff
@@ -391,36 +363,6 @@ https://58bd5b7a98e04.streamlock.net/medienasa-live/_definst_/mp4:RFH_high/playl
 https://586fb512206e4.streamlock.net/rheinmaintv-live/_definst_/mp4:livestream/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RMC
 https://rmc2hdslive-lh.akamaihd.net/i/DVMR_RMC@167269/master.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH 1
-http://79.106.48.2:4578/live/rtsh_1mob_pp2/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH 2
-http://79.106.48.2:4578/live/rtsh_2mob_p2/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH 24
-http://79.106.48.2:4578/live/rtsh_24_mob/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH 3
-http://79.106.48.2:4578/live/rtsh3mobp1/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH Film
-http://79.106.48.2:4578/live/rtsh_film_mob_p1/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH Gjirokastra
-http://79.106.48.2:4578/live/rtsh_gjirokastra_mob_p1/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH Kids
-http://79.106.48.2:4578/live/rtsh_femije_mob_p1/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH Korça
-http://79.106.48.2:4578/live/rtsh_korca_mob_p1/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH Kukesi
-http://79.106.48.2:4578/live/rtsh_kukesi_mob_p1/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH Muzike
-http://79.106.48.2:4578/live/rtsh_muzike_mob_p1/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH Plus
-http://79.106.48.2:4578/live/rtsh_plus_mob_p1/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH Sat
-http://79.106.48.2:4578/live/rtsh_sat_mob_p1/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH Shkodra
-http://79.106.48.2:4578/live/rtsh_shkodra_mob_p1/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH Shqip
-http://79.106.48.2:4578/live/rtsh_shqip_mob_p1/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH Sport
-http://79.106.48.2:4578/live/sport_mob_ppp1/playlist.m3u8
```

Comment:
The links from 79.106.48.2 are currently working
![vlcsnap-2021-09-30-22h04m10s573](https://user-images.githubusercontent.com/7253922/135515440-6508af9e-4258-4db8-97fb-0979a61e6b17.png)

Reply:
But the channels are determined

Reply:
I'm sorry, but I don't understand what you mean.

Reply:
They might work only for certain IPs, doesn't work for me from where I am right now.

Reply:
The channels is same as in al.m3u, so I remove them

Reply:
@taodicakhia We try to keep all versions of the channel broadcast in case one of them fails.

Reply:
@freearhey ok